components/event-bus/Gopkg.toml

```diff
@@ -37,26 +37,31 @@ required = [
   name = "github.com/stretchr/testify"
   version = "1.2.1"
 
-[[constraint]]
+[[override]]
```

Comment:
Is the impact on the existing event bus applications checked?

Reply:
I've tested it and this was the only way to get it running. Marco is doing now a complete integration of it into the new Publish/KN.

Reply:
This indirectly updates the docker artifacts for publish, push and sv by virtue of using different versions of dependencies. Post PR merge, we will have new docker images. We should update the helm charts to use those new images and also ensure eventing e2e works.